scripts/ci/ci_before_install.sh

```diff
@@ -33,6 +33,7 @@ KUBERNETES_MODE=${KUBERNETES_MODE:=""}
 
 mkdir -p "${AIRFLOW_SOURCES}/files"
 
-sudo pip install pre-commit
+# We need to install six apparently
```

Comment:
```suggestion
# We need newer version of six for Travis as they bundle 1.11.0 version
```

Reply:
LGTM, but keep in mind that we need to change this comment when we move to GitHub Actions / remove this commit entirely ;)